src/main/java/com/jvm_bloggers/kafka/serialization/MessageSerializer.java

```diff
@@ -0,0 +1,25 @@
+package com.jvm_bloggers.kafka.serialization;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.jvm_bloggers.kafka.exception.UnableToSerializeMessageException;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class MessageSerializer {
+    private final ObjectMapper objectMapper;
+
+    public MessageSerializer() {
+        objectMapper = new ObjectMapper();
```

Comment:
The way Java's akka streams makes it very unreadable. For example the type of testProbe is actually  ``` TestSubscriber.Probe<ConsumerMessage.CommittableMessage<byte[], String>>``` :D. Before comminting I spoke to Tomek about this issue and we agreed that we make an exception and leave it with def.

This is how it'd look with types:

```java
ConsumerSettings<byte[], String> consumerSettings = ConsumerSettings
                    .create(actorSystem, new ByteArrayDeserializer(), new StringDeserializer())
                    .withBootstrapServers("127.0.0.1:9092")
                    .withGroupId("kafkaTestGroup")
                    .withProperty(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
            NewIssuePublishedMessage publishedMessage = new NewIssuePublishedMessage(15L, "http://jvm-bloggers.com/issue/15");
            TestSubscriber.Probe<ConsumerMessage.CommittableMessage<byte[], String>> testProbe = Consumer.
                    committableSource(consumerSettings, Subscriptions.topics("com.jvm_bloggers.issue.published"))
                    .toMat(TestSink.probe(actorSystem), Keep.right())
                    .run(actorMaterializer);
```